### ISSUE TITLE
Unable to load all geometry

### ISSUE THREAD

**Issue by sinedie** [IFCjs/web-ifc#247]
I'm trying to load a building from a structural analysis software, but IFCBEAMS objects wont load nor show. I'm using ifc-three with react-three-fiber and webworkers enabled. I tried reading the file with ifcopenshell and it read all entities. Seems to be something with the profile definition (IFCDERIVEDPROFILEDEF or IFCSTRUCTURALPROFILEPROPERTIES, idk, I'm new to BIM and IFC world). Also when opening and exporting the same file with FreeCAD it starts to work. Any help would be appreciated. Thank you 

What I get without editing the file (I kind of optimize with python-ifcopenshell, but it is the same issue even if i dont optimize it)
![image](https://user-images.githubusercontent.com/43920544/197877753-335c7af8-f792-45a7-ac32-76aea44251ee.png)

Expected (exported from FreeCad)
![image](https://user-images.githubusercontent.com/43920544/197877854-4546f51e-8898-45ed-98b1-897b810e57b0.png)

[test_files.zip](https://github.com/IFCjs/web-ifc/files/9863930/test_files.zip)

**Comment by QuimMoya** [comment]
Yes, it seems that the second model is using IFCDERIVEDPROFILEDEF, and it was not implemented ... I'm working on it.

**Comment by QuimMoya** [comment]
Ok, test_wrong.ifc now works. Soon i will upload the changes
![image](https://user-images.githubusercontent.com/12625508/199478524-260bfb57-8c8d-47b1-b22a-bea453022040.png)

**Comment by sinedie** [comment]
@QuimMoya You rock. That was too fast.. I guess I have to compile the wasm files and ready to go.

This is fixed, so im gonna close the issue. Ty again.
